gensrc/thrift/AgentService.thrift

```diff
@@ -122,6 +122,22 @@ struct TCreateTabletReq {
     20: optional bool create_schema_file = true;
 }
 
+struct TCreateTableReq {
+    1: optional list<TCreateTabletReq> create_tablet_reqs;
+    2: optional i32 timeout = 0; // unit: second
+}
+
+struct TAbortTxnReq {
+    1: optional Types.TTransactionId txn_id;
+}
+
+struct TReq {
```

Comment:
After offline discussion, should add comments to clarify the necessity of adding a new req framework, including,
1. Why the original task framework cannot work for this situation.
2. What does the txn_id mean, besides for creating table and other ddl operations, is it compatible with data loading process and can be reused by it.